src/Microsoft.ML.Parquet/ParquetLoader.cs

```diff
@@ -18,6 +19,7 @@
 using Parquet;
 using Parquet.Data;
 using Parquet.File.Values.Primitives;
+using DataViewSchema = Microsoft.Data.DataView.Schema;
```

Comment:
The idea was broached I know, but resolution never reached: should we rename? Or else perhaps make `Schema` a nested class of something appropriate, so as to give it definition?

I guess if something as useful as the Parquet format can get away with naming something "schema" without complaint, maybe we (being, I suppose, less universally beloved for now) can get away with it as well fairly easily.

Reply:
We could rename the class to DataViewSchema, or ViewSchema.

We’d probably want to rename everything then. Row is a pretty generic name. So is ColumnType.